src/HrManagement.Identity/Quickstart/Device/DeviceController.cs

```diff
@@ -143,18 +141,15 @@ private async Task<ProcessConsentResult> ProcessConsent(DeviceAuthorizationInput
             return result;
         }
 
-        private async Task<DeviceAuthorizationViewModel> BuildViewModelAsync(string userCode, DeviceAuthorizationInputModel model = null)
+        private async Task<DeviceAuthorizationViewModel> BuildViewModelAsync(string userCode,
+            ConsentInputModel model = null)
```

Comment:
Why did the `DeviceAuthorizationInputModel` transform into `ConsentInputModel` ?